test/jdk/ProblemList.txt

```diff
@@ -785,7 +785,7 @@ java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter_2.java 7131438,802
 java/awt/Modal/WsDisabledStyle/CloseBlocker/CloseBlocker.java 7187741 linux-all,macosx-all
 java/awt/xembed/server/TestXEmbedServerJava.java 8001150,8004031 generic-all
 java/awt/Modal/PrintDialogsTest/PrintDialogsTest.java 8068378 generic-all
-java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.html 8080185 macosx-all,linux-all
+java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.java 8080185 macosx-all,linux-all
```

Comment:
@prsadhuk While looking into the test history I noticed you had previous worked on it PR https://github.com/openjdk/jdk/pull/3406, it wasn't integrated though. Any thoughts whether to make this test windows-only or extend the test to other platforms ?

Reply:
The reason it was not integrated was because of this [comment](https://github.com/openjdk/jdk/pull/3406#pullrequestreview-632106612) wherein it was mentioned to update the test to run without MSOutlook dependancy so that it could be run on all platforms so making it not platform dependant..

Reply:
@prsadhuk Since the test still retains the MS Outlook requirement in instructions, should we make the test windows-only then ? Because having a combination of `@requires (os.family == "windows")` and problem-listing on macOS and linux seems contradicting.

Reply:
@prsadhuk When I read it for the 2nd time it makes sense why the @requires check was added. Since it is a manual test there is no way of knowing it has been problem-listed on macOS & linux unless someone explicitly checks the problemlist before running the test.

Reply:
The @requires was added in [JDK-7147083](https://bugs.openjdk.org/browse/JDK-7147083) to ensure the test pass automatically in non-windows platform much before the problemlist was added so I think it was an oversight and we should remove the problemlist and close 8080185 as NA, in my opinion...